src/api/views.py

```diff
@@ -116,6 +117,15 @@ def patch(self, request, repo_base, repo_name, format=None):
                         status=status.HTTP_200_OK)
 
 
+class ReposPublic(APIView):
+    """
+    Repos that have been made public
+    """
+    def get(self, request, format=None):
+        serializer = RepoSerializer('dh_anonymous', None, request)
```

Comment:
Use settings.ANONYMOUS_ROLE instead of hardcoding 'dh_anonymous' string. Same for https://github.com/datahuborg/datahub/blob/feature/rest_public_tables/src/browser/templates/layout.html#L89.